integrationtest/common/TestCase.d

```diff
@@ -82,7 +88,7 @@ class TestCase
         this.bin = readRequiredOption!string("bin", "Binary directory");
         this.data = test_file[0 .. $-"main.d".length] ~ "data";
         this.common_data = __FILE_FULL_PATH__[0 .. $-"TestCase.d".length] ~ "data";
-        this.git = this.tmp ~ "/" ~ GitPath;
```

Comment:
I think ideally this `this.tmp` should be created inside the directory passed as `tmp` via the option with a random name, i.e. do what `mktemp --tmpdir=DIR --directory` does.